.gitignore

```diff
@@ -0,0 +1 @@
+.stack-work
```

Comment:
Might be a good idea to add .cabal-sandbox too. This way both cabal and stack users are happy :)